core/src/main/java/com/novoda/noplayer/internal/exoplayer/SimpleRenderersFactory.java

```diff
@@ -162,15 +166,31 @@ private void buildVideoRenderers(Context context,
                                      VideoRendererEventListener eventListener,
                                      @ExtensionRendererMode int extensionRendererMode,
                                      List<Renderer> outRenderers) {
-        outRenderers.add(new MediaCodecVideoRenderer(context,
-                mediaCodecSelector,
-                allowedVideoJoiningTimeMs,
-                drmSessionManager,
-                DO_NOT_PLAY_CLEAR_SAMPLES_WITHOUT_KEYS,
-                ENABLE_DECODER_FALLBACK,
-                eventHandler,
-                eventListener,
-                MAX_DROPPED_VIDEO_FRAME_COUNT_TO_NOTIFY));
+        if (allowFallbackDecoder) {
+            outRenderers.add(new MediaCodecVideoRendererWithSimplifiedDrmRequirement(
```

Comment:
When we allow fallbacks: 

- We relax the DRM rules in the renderer so that it will require secure codecs for all `Format`s that have `DrmInitData` regardless of what it is. 
- We use the `MediaCodecSelector` that is capable of providing `secure` and `non-secure` codecs.